ansible/roles/test/files/ptftests/mtu_test.py

```diff
@@ -103,6 +106,52 @@ def check_icmp_mtu(self):
 
     #---------------------------------------------------------------------
 
+    def check_icmpv6_mtu(self):
```

Comment:
The code of ```check_icmpv6_mtu``` is similar to ```check_icmp_mtu```. Can you try to refactor the code by adding a param fixture ```ipver``` and reuse the current code?

Reply:
building packets and masking packets are different, sending and receiving can be reused, Thank you!